packages/fxa-payments-server/src/lib/fluent-react.d.ts

```diff
@@ -0,0 +1 @@
+declare module '@fluent/react';
```

Comment:
I think you should be able to use https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/types/fluent__react/.